Doc/library/typing.rst

```diff
@@ -1005,5 +1005,10 @@ The module defines the following classes, functions and decorators:
       if TYPE_CHECKING:
           import expensive_mod
 
-      def fun():
-          local_var: expensive_mod.some_type = other_fun()
+      def fun(arg: 'expensive_mod.SomeType') -> None:
+          local_var: expensive_mod.AnotherType = other_fun()
+
+   (Note that the first type annotation must be enclosed in quotes, making it a
+   "forward reference", to hide the expensive_mod reference from the
+   interpreter runtime. Type annotations for local variables are not evaluated,
```

Comment:
Please add two spaces after a full stop. See http://cpython-devguide.readthedocs.io/documenting.html#use-of-whitespace for details.